dev/devicelab/bin/tasks/technical_debt__cost.dart

```diff
@@ -25,12 +26,15 @@ Future<double> findCostsForFile(File file) async {
       path.extension(file.path) != '.yaml' &&
       path.extension(file.path) != '.sh')
     return 0.0;
+  final bool isTest = file.path.endsWith('_test.dart');
   double total = 0.0;
   for (String line in await file.readAsLines()) {
     if (line.contains(todoPattern))
       total += todoCost;
     if (line.contains(ignorePattern))
       total += ignoreCost;
+    if (isTest && line.contains('skip:'))
```

Comment:
In addition to this, tests could also be marked as skipped with the `@Skip` annotation: https://pub.dartlang.org/packages/test#skipping-tests